distributions/openhab-addons/pom.xml

```diff
@@ -12,16 +12,44 @@
     <artifactId>openhab-addons</artifactId>
     <packaging>kar</packaging>
 
-    <name>openHAB Add-ons KAR</name>
```

Comment:
I changed the name because it is not only "the KAR" but also the feature repository XML used in the distribution. WDYT?

Reply:
yeah, makes sense.